torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -775,18 +775,25 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
             if node.target == torch.ops.aten.split.Tensor:
                 # convert input tensopr to ROW MAJOR layout for split
                 to_layout = g.call_function(ttnn.to_layout, (args[0],), {"layout": TtnnRowMajorLayout()})
+                if len(args[0].meta["val"].size()) == 1:
+                    # For example, the input shape original is [768]
+                    # But due to issue #390 it become [1, 768] and cause failed
+                    # remove this part once #390 is solved
+                    return None
 
                 # convert relative split dim to absolute
-                if args[2] >= 0:
-                    split_dim = args[0]
+                dim = args[2] if len(args) > 2 else 0
+                if dim >= 0:
+                    split_dim = dim
                 else:
-                    split_dim = len(args[0].meta["val"].size()) + args[2]
+                    split_dim = len(args[0].meta["val"].size()) + dim
 
                 # convert from PyTorch size of chunk to ttnn number of chunks
                 if isinstance(args[1], int):
                     num_chunks = math.floor(args[0].meta["val"].size()[split_dim] / args[1])
                 else:
-                    raise RuntimeError(f"ttnn.split only supports chunks of same size.")
+                    # ttnn.split only supports chunks of same size.
+                    return None
```

Comment:
is there a ticket to fix the gap in tt-metal?

Reply:
I think this issue mentioned?
https://github.com/tenstorrent/tt-metal/issues/12859